src/client/Licit.js

```diff
@@ -615,6 +618,26 @@ class Licit extends React.Component<any, any> {
     view.dispatch(tr.setSelection(TextSelection.atEnd(view.state.doc)).scrollIntoView());
     view.focus();
   };
+  //To insert empty text property for content without text attribute
+  insertTextNodes = (docJson) => {
+    // Create a deep copy of docJson to avoid modifying the original object
+    const deepCopy = JSON.parse(JSON.stringify(docJson));
```

Comment:
use browser native structuredClone instead of sting parsing. Also, cloning should be unnecessary here since the doc isn't loaded yet.